pkg/package-server-manager/controller_test.go

```diff
@@ -72,10 +74,39 @@ func intOrStr(val int) *intstr.IntOrString {
 	return &tmp
 }
 
+type testCSVOption func(*olmv1alpha1.ClusterServiceVersion)
+
+func withVersion(v semver.Version) func(*olmv1alpha1.ClusterServiceVersion) {
+	return func(csv *olmv1alpha1.ClusterServiceVersion) {
+		csv.Spec.Version = version.OperatorVersion{v}
+	}
+}
+
+func withDescription(description string) func(*olmv1alpha1.ClusterServiceVersion) {
+	return func(csv *olmv1alpha1.ClusterServiceVersion) {
+		csv.Spec.Description = description
+	}
+}
+
+func withAffinity(affinity *corev1.Affinity) func(*olmv1alpha1.ClusterServiceVersion) {
+	return func(csv *olmv1alpha1.ClusterServiceVersion) {
+		csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs[0].Spec.Template.Spec.Affinity = affinity
+	}
+}
+func withRolloutStrategy(strategy *appsv1.RollingUpdateDeployment) func(*olmv1alpha1.ClusterServiceVersion) {
```

Comment:
Shoot, @grokspawn could you change this to the name of the field?

Reply:
Maybe withRollingUpdateStrategy? Open to suggestions

Reply:
How about `withRollingUpdateStrategy`?

Reply:
Funny that your comment came in after mine, but at least great minds think alike. :)